### PR TITLE
Miscellaneous fixes for the 'me' command.

### LINKED ISSUE #23: Update Profile Command
Currently just plaintext, usernames can affect Discord formatting.  Display usernames in an inline codeblock.

### PR DESCRIPTION
**Pull request by JacobCallahan**
Also through in a fix to the dockerfile so it correctly loads commands.
Fixes #23

### CODE REVIEW

File: src/commands/profile/Me.ts [hunk 1]
```diff
@@ -41,111 +43,80 @@ export default class Me extends Command<Client>
...
+				if (args[1] && !Constants.platformRegExp.test(args[1])) {
 					error = true;
 					errorMessage += 'Please specify a valid `Platform`.';
 				}
 
-				if (error)
-				{
-					message.channel.send(errorMessage);
+				if (error) {
+					message.channel.sendMessage(errorMessage);
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should be `message.channel.send(errorMessage);`

File: src/commands/profile/Me.ts [hunk 2]
```diff
@@ -41,111 +43,80 @@ export default class Me extends Command<Client>
...
 						error = true;
 						errorMessage += 'Please specify a valid XBL handle. ';
 					}
 				}
-
-				if (error)
-				{
-					message.channel.send(errorMessage);
+				if (error) {
+					message.channel.sendMessage(errorMessage);
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should be `message.channel.send(errorMessage);`

File: src/commands/profile/Me.ts [hunk 3]
```diff
@@ -41,111 +43,80 @@ export default class Me extends Command<Client>
...
-					{
+					const setFilter: any = (m: Message) => {
 						if (m.author.id === message.author.id && m.content.match(/r|a/i))
 							return true;
 					};
 
 					// send confirmation message
-					message.channel.send(`You appear to have an active Handle already.\n\n[\`R\`]eplace account?\n[\`A\`]dd account?`).then(() =>
-					{
+					message.channel.sendMessage(`You appear to have an active handle already.\n\n[\`R\`]eplace account?\n[\`A\`]dd account?`).then(() => {
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should be `message.channel.send`

File: src/commands/profile/Me.ts [hunk 4]
```diff
@@ -267,26 +230,23 @@ export default class Me extends Command<Client>
 						.setColor(Profile.getEmbedColor(profile.handles[index].platform))
 						.setAuthor(message.author.username, message.author.avatarURL)
 						.addField('Handle', handles, true)
-						.setFooter(`Which of the above would you like to activate?`);
+						.setFooter(`Which number above would you like to activate?`);
 
 					// display prompt
-					message.channel.send({ embed: embed }).then(() =>
-					{
+					message.channel.sendEmbed(embed, '', { disableEveryone: true }).then(() => {
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should be `message.channel.send({ embed: embed }).then(() => {`

File: src/commands/profile/Me.ts [hunk 5]
```diff
@@ -267,26 +230,23 @@ export default class Me extends Command<Client>
...
-						.then((collected: Collection<string, Message>) =>
-						{
+						.then((collected: Collection<string, Message>) => {
 							// grab index from response
 							index = parseInt(collected.first().content) - 1;
 
 							// make sure it's within range
 							if (index > profile.handles.length - 1)
-								return message.channel.send('That is not a valid selection.');
+								return message.channel.sendMessage('Please respond only one of the numbers listed.');
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should be `message.channel.send()`

File: src/commands/profile/Me.ts [hunk 6]
```diff
@@ -301,27 +261,24 @@ export default class Me extends Command<Client>
...
 					message.channel.send('No profile attributes set.');
 					return message.channel.stopTyping();
 				}
 
 			default:
 				// check if valid profile
-				if (profile)
-				{
+				if(args[0]) {
+					message.channel.sendMessage('Invalid argument. See help for this command.')
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: should use `message.channel.send()`

File: src/commands/profile/Me.ts [hunk 7]
```diff
@@ -257,8 +221,7 @@ export default class Me extends Command<Client>
 					profile.handles.sort((a: Handle, b: Handle) => { return (a.active === b.active) ? 0 : a.active ? -1 : 1; });
 
 					// create list of handles for display
-					for (let x: number = 0; x < profile.handles.length; x++)
-					{
+					for (let x: number = 0; x < profile.handles.length; x++) {
 						handles += (x + 1) + `⃣  ${profile.handles[x].tag} \`${profile.handles[x].platform.toUpperCase()}\`\n`;
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: need to wrap `${profile.handles[x].tag}` in backticks.

File: src/commands/profile/Me.ts [hunk 8]
```diff
@@ -257,36 +221,32 @@ export default class Me extends Command<Client>
...
 						.setColor(Profile.getEmbedColor(profile.handles[index].platform))
 						.setAuthor(message.author.username, message.author.avatarURL)
 						.addField('Handle', handles, true)
-						.setFooter(`Which of the above would you like to activate?`);
+						.setFooter(`Which number above would you like to activate?`);
 
 					// display prompt
-					message.channel.send({ embed: embed }).then(() =>
-					{
+					message.channel.send(embed, '', { disableEveryone: true }).then(() => {
```
Comment (katagatame) [inline src/commands/profile/Me.ts]: change this line to...

`message.channel.send({ embed: embed }).then(() => {`

**Comment by weylin** [comment]
This LGTM.

**Pull request merged**